### PR TITLE
azure: add troubleshooting directions

### PR DESCRIPTION
**Pull request by colemickens**
Add some basic troubleshooting directions for Azure.

### CODE REVIEW

**Comment by colemickens** [comment]
@ahmetalpbalkan could you have a glance at this if you get a chance? You usually have good verbiage feedback for me. Thanks.

File: phase1/azure/README.md [hunk 1]
```diff
@@ -108,6 +108,28 @@ cp ./phase1/azure/.tmp/kubeconfig.json ~/.kube/config
 
 Enjoy your Kubernetes cluster on Azure!
 
+## Troubleshooting
+
+### Validation Fails (Zero nodes are healthy)
+If no nodes are avaiable, there was likely a provisioning failure on the master (either in Azure or in the `ignition` provisioning container).
```
Comment (ahmetalpbalkan) [inline phase1/azure/README.md]: typo:avaiable

**Comment by ahmetalpbalkan** [comment]
looks good.

File: phase1/azure/README.md (hunk 1 above)
Comment (colemickens) [inline phase1/azure/README.md]: Thx.

**Pull request merged**